platform/windows/display_server_windows.cpp

```diff
@@ -2536,7 +2536,7 @@ LRESULT DisplayServerWindows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARA
 		case WM_KEYUP:
 		case WM_KEYDOWN: {
 			if (wParam == VK_SHIFT)
-				shift_mem = uMsg == WM_KEYDOWN;
+				shift_mem = (uMsg == WM_KEYDOWN || uMsg == WM_SYSKEYDOWN);
```

Comment:
Should the same be done for `VK_CONTROL` below?

Reply:
I think you're probably right.  I was only fixing the bug that was mentioned in the issue, but I assume that shortcuts involving control/alt might result in the same bug.  I will make that change and update the pull request.